coalib/parsing/DefaultArgParser.py

```diff
@@ -182,18 +182,25 @@ def default_arg_parser(formatter_class=None):
                                    action='store_const',
                                    help=SHOW_BEARS_HELP)
 
-        outputs_group.add_argument('-A',
-                                   '--show-all-bears',
-                                   const=True,
-                                   action='store_const',
-                                   help="Display all bears.")
-
-        SHOW_LANGUAGE_BEARS = ("Display all bears for the given languages.")
         outputs_group.add_argument('-l',
-                                   '--show-language-bears',
+                                   '--filter-by-language',
                                    nargs='+',
                                    metavar='LANG',
-                                   help=SHOW_LANGUAGE_BEARS)
+                                   help="Filters the output of `--show-bears` "
+                                        "by the given languages.")
```

Comment:
yes, just as before

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala-analyzer.org/ - http://viperdev.io/ - http://gitmate.com/

On 11 June 2016 at 11:28, Adhityaa Chandrasekar <notifications@github.com>
wrote:

> In coalib/parsing/DefaultArgParser.py
> <https://github.com/coala-analyzer/coala/pull/2304#discussion_r66703645>:
>
> >                                     nargs='+',
> >                                     metavar='LANG',
> > -                                   help=SHOW_LANGUAGE_BEARS)
> > +                                   help="Filters the output of `--show-bears` "
> > +                                        "by the given languages.")
>
> *languages* Can multiple languages be given here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/2304/files/5cdc13ffb7fd940e55a56831ea6f3aa1e371d355..e79ae8caa3dbcfe89be83ba52f063d319354818c#r66703645>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFc6KOXbZQDaHbGcKTpkv4Zul5s3Syvgks5qKn-ugaJpZM4Ivpvk>
> .
>